src/btree/bt_vrfy_dsk.c

```diff
@@ -65,11 +66,22 @@ static int __verify_dsk_row_leaf(
  */
 int
 __wt_verify_dsk_image(WT_SESSION_IMPL *session, const char *tag, const WT_PAGE_HEADER *dsk,
-  size_t size, WT_ADDR *addr, bool empty_page_ok)
+  size_t size, WT_ADDR *addr, uint32_t verify_flags)
 {
     uint8_t flags;
     const uint8_t *p, *end;
 
+    /* Initialize the verify information. */
+    WT_VERIFY_INFO vi;
+    vi.session = session;
```

Comment:
Why we add session into the struct? Generally, we always pass session as a separate argument.

Reply:
We thought it might be cleaner to put it in the struct but that may be subjective.